### PR TITLE
Remove ckeditor from packages

### PR DESCRIPTION
**Pull request by Katrienvh**
## Type

- Non critical bugfix

## Pull request description

Ckeditor is not used anymore we forgot to remove the packages, now they are.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,11 +7,6 @@
   "author": "Tijs Verkoyen <tijs@verkoyen.eu>",
   "license": "MIT",
   "devDependencies": {
-    "@editorjs/editorjs": "^2.16.1",
```
Comment (carakas) [inline package.json:10]: Why did you remove this as well?

**Comment by Katrienvh** [comment]
Totally removed the wrong ones, started over here: https://github.com/forkcms/forkcms/pull/3217
